fvm/src/gas/price_list.rs

```diff
@@ -1076,6 +1076,16 @@ impl Rules for WasmGasPrices {
             | Operator::Return
             | Operator::Else
             | Operator::End => Ok(InstructionCost::Fixed(0)),
+            // Make casts and constants free.
+            Operator::I64ExtendI32U // only unsigned is free.
+            | Operator::I32WrapI64 // truncates
+            | Operator::I32ReinterpretF32 // casts
+            | Operator::I64ReinterpretF64 // casts
+            | Operator::V128Const { .. }
```

Comment:
V128 is simd, I'm not sure if we can do simd

Reply:
We don't.... I guess I can just nop-out all SIMD instructions.